website/docs/components/application-state/partials/code/how-to-use.md

```diff
@@ -33,37 +33,49 @@ This component intends to replace a few different simple error and empty/zero st
 </Hds::ApplicationState>
 ```
 
-#### Empty state with a footer link and divider
+#### Empty state with yielded body block
 
 ```handlebars
 <Hds::ApplicationState as |A|>
   <A.Header @title="Empty state title text" />
-  <A.Body @text="The item you were looking for was not found." />
-  <A.Footer @hasDivider={{true}} as |F|>
-    <F.LinkStandalone @icon="help" @text="Need Help" @href="/components/alert"
-    @iconPosition="trailing" />
+  <A.Body>
+    <Doc::Placeholder @text="block yield" @height="100" @background="#eee" />
+  </A.Body>
+  <A.Footer as |F|>
+    <F.LinkStandalone @icon="arrow-left" @text="Go back" @href="/" />
   </A.Footer>
 </Hds::ApplicationState>
 ```
 
-#### Empty state with yielded body block
+#### Empty state with body text
 
 ```handlebars
 <Hds::ApplicationState as |A|>
   <A.Header @title="Empty state title text" />
-  <A.Body>
-    <Doc::Placeholder @text="block yield" @height="100" @background="#eee" />
-  </A.Body>
+  <A.Body @text="Some sentence that conveys a good message to the user" />
   <A.Footer as |F|>
     <F.LinkStandalone @icon="arrow-left" @text="Go back" @href="/" />
   </A.Footer>
 </Hds::ApplicationState>
 ```
 
-#### Empty state with body text
+#### Empty state with center alignment
+
+```handlebars
+<Hds::ApplicationState @align="center" as |A|>
+  <A.Header @title="Empty state title text" />
+  <A.Body @text="Some sentence that conveys a good message to the user" />
+  <A.Footer as |F|>
+    <F.LinkStandalone @icon="arrow-left" @text="Go back" @href="/" />
+  </A.Footer>
+</Hds::ApplicationState>
+```
+
+#### Empty state with media
 
 ```handlebars
 <Hds::ApplicationState as |A|>
+  <A.Media><img src="/assets/images/avatar.png" /></A.Media>
```

Comment:
Could you please add an alt attribute to this `img` tag example?

(All img tags should include an alt attribute. If the image is decorative only then you can leave the alt value empty.)

Examples:
```
<img src="helios-logo.png" alt="Helios" />

<img src="decoration.png" alt="" />
```

Reply:
Please add alt attributes to below img tags also.